src/ng-option-highlight/lib/ng-option-highlight.directive.spec.ts

```diff
@@ -27,16 +30,16 @@ describe('NgOptionHighlightDirective', () => {
         fixture.detectChanges();
     });
 
-    it('should have two elements with highlight directive', () => {
+    it('should have five elements with highlight directive', () => {
         const highlightDirectives = fixture.debugElement.queryAll(By.directive(NgOptionHighlightDirective));
-        expect(highlightDirectives.length).toBe(2);
+        expect(highlightDirectives.length).toBe(5);
     });
 
     it('should have one element with highlighted text when term matching', () => {
         const span = fixture.debugElement.query(By.css('#test1'));
-        fixture.componentInstance.term = 'is high';
+        fixture.componentInstance.term = 'highlighted';
         fixture.detectChanges();
-        expect(span.nativeElement.querySelector('.highlighted').innerHTML).toBe('is high');
+        expect(span.nativeElement.querySelector('.highlighted').innerHTML).toBe('highlighted');
```

Comment:
why do you change the test case?

Reply:
My code changes the behavior slightly, in the case of the search `is high` against `this text is highlighted`, it would highlight each word separately `this text <highlighted>is</highlighted> <highlighted>high</highlighted>lighted` instead of `this text <highlighted>is high</highlighted>lighted`

I can change the test case to that string `is high` and change the expected result

Reply:
I would suggest to keep the test case and adjust the expected result accordingly

Reply:
Alright, updated for the new behavior!